testing-standard.md

```diff
@@ -177,10 +177,42 @@ There are three things we should check for in the above function:
  2. That we retrieve the deathHours.
  3. That we correctly calculate the hours alive based on the two.
 
-By stubbing `getHoursAtBirth` and `getHoursAtDeath` we can **a)** check that both of these are called (ticking off points 1 and 2 above), and **b)** ensure that the real versions of `getHoursAtBirth` and `getHoursAtDeath` are never actually called. This is crucially important as we're **not testing that those functions work**. We only care that they've been called and that what we do in the `hoursAlive` function works. 
+By stubbing `getHoursAtBirth` and `getHoursAtDeath` we can **a)** check that both of these are called (ticking off points 1 and 2 above), and **b)** ensure that the real versions of `getHoursAtBirth` and `getHoursAtDeath` are never actually called. This is crucially important as we're **not testing that those functions work**. We only care that they've been called and that what we do in the `hoursAlive` function works.
 
 Other tests will check that `getHoursAtBirth` and `getHoursAtDeath` work, and by building up a suite of tests that ensure each function works as it's supposed to, we can build confidence that our codebase works correctly. It's for this reason that `stubs` are preferred to spies in our unit tests - we don't want to be running code outside of the function we're actually testing during a unit test.
 
+### Sinon Sandbox
+
+Using **sinon.sandbox** object to hold your stubs and spies.
+
+Benefits:
+
+Allows you to be able to put multiple stubs within an object when creating them in a **beforeEach** and instead of restoring them one at a time can restore them all at once.
+This is nice because it means we can restore all of our stubs at once and removes a few lines of code.
+
+Code:
+Put the creation of your sandbox before the beforeEach because then your **afterEach** has scope to what is in your **beforeEach**.
+
```

Comment:
don't think you need the beforeEach/afterEach explanation as devs already know this.

Same as on line 214